pdr_backend/pdr_dashboard/test/test_callbacks_feeds.py

```diff
@@ -0,0 +1,231 @@
+import json
+import time
+
+from selenium.webdriver.common.by import By
+from pdr_backend.pdr_dashboard.test.resources import (
+    _navigate_to_feeds_page,
+    _assert_table_row_count,
+    start_server_and_wait,
+    _clear_filters,
+    _set_dropdown_and_verify_row_count,
+    _set_input_value_and_submit,
+    _remove_tags,
+    _set_searchbar_value,
+)
+
+
+def _get_table_data_as_json(table):
```

Comment:
This function doesn't return a json, it returns a dictionaty as far as I can tell. Please adjust its name.